news/2017-10-06-scorewatch-september-week-3-4.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "Scorewatch: September Week 3 & 4"
+date: 2017-10-06 15:00:00 +0200
+---
+
+We got another 2 weeks in 1 bundle for this Scorewatch article, but plays have been dropping quickly! We got some Chocobo goodness, even more unbelievable 700pp plays and a Bikko showcase. Read on to find out more!
+
+[Toy](https://osu.ppy.sh/u/2757689) is off to make some more insane plays, this time on [Hyadain - Chocobo](https://osu.ppy.sh/b/24023) as he takes the first ever HDHR SS on the map to earn himself 247pp! AR7 and CS7.8 is nothing to this maestro.
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/WSrqfAi3Liw" frameborder="0" allowfullscreen></iframe></br>
+
+[idke](https://osu.ppy.sh/u/4650315) is the next player to take the prestigious 700pp score, and doing it in some crazy fashion by snatching the first FC on [BABYMETAL - Road of Resistance](https://osu.ppy.sh/b/1201636) with 99.23% to gain 718pp! Who will be the next to take the 700pp medal? Time will tell.
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/UMh_nT3gIoY" frameborder="0" allowfullscreen></iframe></br>
+
+Remember Hollow Wings’ famous [Halozy - Kikoku Doukoku Jigokuraku](https://osu.ppy.sh/b/942356) map? Well, it seems like the champion above champions has stepped up and challenged this slider-hell map, setting an incredible 99.94% HDSD FC to take the number one spot on the map and 216pp. What is next for the Norwegian? 
```

Comment:
capitalize Champion Above Champions if you're going to omit his name so it's more obvious